### ISSUE TITLE
Skip in Music Trivia Will not work

### ISSUE THREAD

**Issue by DevEvelynn** [galnir/Master-Bot#620]
Do not open an issue if you simply "copied" code over to your bot/another bot. This is absolutely not recommended and will cause bugs. Also do not open an issue if you modified code and added features and now it's not working right. This is because I can't figure it out and don't have the time to read your code and find out what you did wrong.

**Describe the bug**
Skip won't work for music trivia
**To Reproduce**
1. Use 'x' command
2. provide 'y' argument

**Expected behavior**
It to skip the song
**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows, Ubuntu...] Pterodactyl
 - Node Version(Should be v14 atleast): 16.9.1
 - Is python 2.7 installed?: yes
 - How are you hosting the bot(Locally, on a vps, heroku, glitch...):  VPS

**Additional context**
It will not work

**Comment by DevEvelynn** [comment]
Also The Lyrics Command Will not Work

**Comment by galnir** [comment]
1. Lyrics command works, regularly tested on different machines. Maybe your API key is invalid?
2. Skip works in trivia, also tested. You can skip a song by typing skip (will skip depends on how many people participate in the trivia and also type skip) or eliminate it by using the 'stop-trivia' command

**Comment by DevEvelynn** [comment]
> 
> 
>     1. Lyrics command works, regularly tested on different machines. Maybe your API key is invalid?
> 
>     2. Skip works in trivia, also tested. You can skip a song by typing skip (will skip depends on how many people participate in the trivia and also type skip) or eliminate it by using the 'stop-trivia' command

Basicly I think the reason i thought it didnt work was because theres no way of telling whos skipped

**Comment by DevEvelynn** [comment]
also whenever i put name & artist together thats correct the bot crashes

**Comment by DevEvelynn** [comment]
![image](https://user-images.githubusercontent.com/45043402/133232557-bdbaba37-4543-46a5-8c8d-6f7857132a15.png)
```javascript
/home/container/utils/music/TriviaPlayer.js:149
              this.score(msg.author.username) + 2
                   ^

TypeError: this.score is not a function
    at MessageCollector.<anonymous> (/home/container/utils/music/TriviaPlayer.js:149:20)
    at MessageCollector.emit (node:events:394:28)
    at MessageCollector.handleCollect (/home/container/node_modules/discord.js/src/structures/interfaces/Collector.js:108:12)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```
container@pterodactyl~ Server marked as offline...

**Comment by galnir** [comment]
fixed in 584dc34
thanks for notifying me about bugs @DevEvelynn :)

**Comment by DevEvelynn** [comment]
> 
> 
> fixed in [584dc34](https://github.com/galnir/Master-Bot/commit/584dc3423a0b3ab630cc43227b64ddfbb3668771)
> thanks for notifying me about bugs @DevEvelynn :)

No Problem
